tutorials/airports/Tutorial.md

```diff
@@ -299,7 +299,7 @@ Once we've generated outline paths for each state, plotting them is straightforw
 
 Here we add a definition for a `path` mark to the `marks` array. We use the `layout_path` property for the path outline, set the `fill` color to light grey, and set the `stroke` color to white.
 
-```!vega
+```!vega(num[0-3600] scale=1200, num[0-900] cx=450, num[0-560] cy=280)
```

Comment:
We should explore the syntax possibilities here... perhaps consider a JSON format for easy parsing and re-use elsewhere? (If not too clunky...)